setup.py

```diff
@@ -15,7 +15,7 @@
 """
 from __future__ import division, print_function
 
-DOCLINES = __doc__.split("\n")
+DOCLINES = (__doc__ or '').split("\n")
```

Comment:
The value of `(__doc__ or '')` depends on non-obvious properties of `or`, e.g., that is returns the first true value rather than a boolean. How about `DOCLINES = __doc__ .split("\n") if __doc__ else ''` Which might also be a bit obscure in that the first expression is not evaluated unless `__doc__` is true, but at least acts like c.